server.js

```diff
@@ -542,7 +578,29 @@ app.post('/userupload', upload.single('usercsv'), function(req, res) {
  */
 app.post('/questionadd', function(req, res) {
     req.body.basePoints = parseInt(req.body.basePoints);
-    req.body.type = questions.QUESTION_REGULAR;
+    // correct the question Type in database:
+    switch(req.body.type){
+        case 're':
```

Comment:
can we have some documentation for the naming

Reply:
The naming convention was already there in server/question.js